frontend/src/components/Details/MessageHEX.vue

```diff
@@ -1,6 +1,8 @@
 <template>
 	<div class="w-full">
-		<div>Message (HEX):</div>
+		<div>Message (HEX):
+			<InfoTooltip text="Input parameters given to activity entrypoint. TODO (maybe something with HEX)"/>
```

Comment:
Yes, the same as for logs. But please don't use "HEX" in all-caps. It's not an acronym.